test/average.cpp

```diff
@@ -4,50 +4,51 @@
 #include "test/compare.hpp"
 
 
-using namespace dr;
-
 int main(int argc, char * * argv) {
 	testing::InitGoogleTest(&argc, argv);
 	return RUN_ALL_TESTS();
 }
 
-TEST(AverageQuaternionTest, averageOrientations) {
+namespace dr {
+
+TEST(AverageQuaternionTest, averageOrientations) { //NOLINT
 	// Define orientations
-	Eigen::Matrix3d m1, m2;
-	m1 = Eigen::AngleAxisd(0.25*M_PI, Eigen::Vector3d::UnitX());
-	m2 = Eigen::AngleAxisd(0.75*M_PI, Eigen::Vector3d::UnitX());
+	Eigen::Matrix3d m1;
+	Eigen::Matrix3d m2;
+	m1 = Eigen::AngleAxisd(0.25*M_PI, Eigen::Vector3d::UnitX()); //NOLINT
+	m2 = Eigen::AngleAxisd(0.75*M_PI, Eigen::Vector3d::UnitX()); //NOLINT
 
 	// Convert to quaternions
 	std::vector<Eigen::Quaterniond> orientations;
-	orientations.push_back( Eigen::Quaterniond(m1) );
-	orientations.push_back( Eigen::Quaterniond(m2) );
+	orientations.emplace_back( Eigen::Quaterniond(m1) );
```

Comment:
This should be `orientations.emplace_back( m1 );`, and analogue for the other cases of replacing push_back with emplace_back